Entity/NewsItem.php

```diff
@@ -2,24 +2,71 @@
 
 namespace Example\NewsBundle\Entity;
 
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use JMS\Serializer\Annotation as Serializer;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use JMS\Serializer\Annotation\Expose;
+use JMS\Serializer\Annotation\VirtualProperty;
+use Sulu\Bundle\MediaBundle\Entity\Media;
+
+/**
+ * @ExclusionPolicy("all")
```

Comment:
this annotation will be used by http://jmsyst.com/libs/serializer to serialize instances of this class. for a full list of supported annotations look at this list http://jmsyst.com/libs/serializer/master/reference/annotations